package.json

```diff
@@ -6,6 +6,7 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
+    "bcrypt": "^3.0.6",
```

Comment:
When I tried to build this image locally, it failed. It appears bcrypt [depends on having Python](https://www.npmjs.com/package/bcrypt#user-content-dependencies), which is not included in the alpine-based container we're using for Docker. Changing `Dockerfile` contents to this worked for me:
```
FROM node:8-jessie

COPY . /app

WORKDIR /app

RUN apt-get update && \
  npm install --only=production && \
  chmod a+x /app/docker/entrypoint.sh

EXPOSE 3000

CMD ["/app/docker/entrypoint.sh"]
```